src/frontend/src/tests/pages/ChangeRequestsPage/ChangeRequestsTable.test.tsx

```diff
@@ -40,6 +43,13 @@ const mockHook = (isLoading: boolean, isError: boolean, data?: ChangeRequest[],
   mockTheme.mockReturnValue(themes[0]);
 };
 
+jest.mock('../../../../hooks/auth.hooks');
+const mockedUseAuth = useAuth as jest.Mock<Auth>;
+
+const mockAuthHook = (user = exampleAdminUser) => {
```

Comment:
you can use my new testing method here. search the whol eproject for `authHooks` and copy how i do it there or look at that pr i made

Reply:
TypeError: Cannot destructure property 'isLoading' of '(0 , _changeRequests.useAllChangeRequests)(...)' as 
it is undefined.

      118 |   const [state, setState] = useState<number[]>([]);
      119 |   const [implemented, setImplemented] = useState('');
    > 120 |   const { isLoading, isError, data, error } = useAllChangeRequests();

Reply:
getting this error when I do that